pkg/server/server.go

```diff
@@ -474,47 +485,68 @@ func (s *ProxyServer) Connect(stream agent.AgentService_ConnectServer) error {
 
 // route the packet back to the correct client
 func (s *ProxyServer) serveRecvBackend(stream agent.AgentService_ConnectServer, agentID string, recvCh <-chan *client.Packet) {
+	defer func() {
+		// Close all connected frontends when the agent connection is closed
+		// TODO(#126): Frontends in PendingDial state that have not been added to the
+		//             list of frontends should also be closed.
+		frontends, _ := s.getFrontends(agentID)
+		klog.Infof("<<< Close %d frontends connected to agent %s", len(frontends), agentID)
```

Comment:
Not worth holding. This log message seems more important than a data sent message. We should explore coming up with a sane log level strategy, which has something like state transitions (things like this a close connection) at a more significant log level than business as usual messages (sent a packet)